modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -144,36 +234,284 @@ function inope_policy_review_form($form, &$form_state, $node) {
 
 /**
  * The iNope Policy Review Form submit handler.
+ *
+ * This submit handler is broken up into 5 parts.
+ * PART 1 - Set variables and load necessary objects.
+ * PART 2 - Update existing entries if policy review id is a non empty string.
+ * PART 3 - Insert an approved response.
+ * PART 4 - Insert a changes requested response.
+ * PART 5 - Catch errors.
  */
 function inope_policy_review_form_submit($form, &$form_state) {
-  // Load objects from which their properties will be displayed.
+  // PART 1 - Load objects from which their properties will be displayed.
   $account = user_load($form_state['values']['inope_policy_review_hidden_uid']);
   $node = node_load($form_state['values']['inope_policy_review_hidden_nid']);
+  $text = check_plain($form_state['values']['inope_policy_review_changes_requested_textarea']);
 
-  // If 'I have reviewed this policy and approved it for use' is selected.
-  if ($form_state['values']['inope_policy_review_status_radio'] == 0) {
-    drupal_set_message(t('Policy approved for use'));
-    // TODO: put this into the database.
-  }
-  // If 'I have reviewed this policy and would like to see changes' is selected.
-  elseif ($form_state['values']['inope_policy_review_status_radio'] == 1) {
+  // PART 2 - Check to see if we are updating an existing entry by checking if
```

Comment:
See note in other PR -- i think this was a bit of confusion, we are doing only inserts, and preserving history, they will only be able to change an answer on a current vid.